source/slang/slang-capabilities.capdef

```diff
@@ -317,6 +317,7 @@ def _GL_ARB_sparse_texture_clamp : _GL_ARB_sparse_texture2;
 def _GL_ARB_texture_gather : _GLSL_130;
 def _GL_ARB_texture_query_levels : _GLSL_130;
 def _GL_ARB_shader_clock : _GLSL_450;
+def _GL_ARB_gpu_shader_int64 : _GLSL_450;
```

Comment:
[this should be _GLSL_400, GLSL_400 is the requirement to use the extension](https://registry.khronos.org/OpenGL/extensions/ARB/ARB_gpu_shader_int64.txt)